CITATION.cff

```diff
@@ -0,0 +1,82 @@
+cff-version: 1.2.0
+title: The Swedish Reference Genome Portal
+message: "If you use or reuse this software, please cite it using the following metadata."
+type: software
+authors:
+  - family-names: Lantz
+    given-names: Henrik
+  - family-names: Brink
+    given-names: Daniel P.
+    orcid: "https://orcid.org/0000-0003-4041-0250"
+  - family-names: Crean
+    given-names: Rory
+  - family-names: Ågren
+    given-names: Quentin
+  - family-names: Fuentes-Pardo
+    given-names: Angela P. 
+    orcid: "https://orcid.org/0000-0002-5734-9030"
+  - family-names: Kochari
+    given-names: Arnold
+    orcid: "https://orcid.org/0000-0003-1373-5121"
+  - family-names: Kutima
+    given-names: Hanna
+    orcid: "https://orcid.org/0000-0001-7724-2567"
+  - family-names: Rung
+    given-names: Johan
+    orcid: "https://orcid.org/0000-0001-5875-8429"
+  - family-names: Persson
+    given-names: Bengt
+repository-code: "https://github.com/ScilifelabDataCentre/genome-portal"
+url: "https://genomes.scilifelab.se"
+abstract: "The Swedish Reference Genome Portal is a platform facilitating access and discovery of genome data of non-model eukaryotic species studied in Sweden."
+license: MIT
+references:
+  - authors:
+      - family-names: Diesh
```

Comment:
I wonder if inserting the whole author list is necessary 🤔 According to JBrowse2 [citations instructions](https://jbrowse.org/jb2/), article and DOI should be satisfactory.